### PR TITLE
docs: edgy bedrock.

### PR DESCRIPTION
**Pull request by thorwebdev**

### CODE REVIEW

**Review by dijonmusters — APPROVED** [review]
Looks great! 👍

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts [hunk 1]
```diff
@@ -1171,6 +1171,10 @@ export const functions: NavMenuConstant = {
           name: 'Sending Push Notifications',
           url: '/guides/functions/examples/push-notifications',
         },
+        {
+          name: 'Generating AI images ',
```
Comment (charislam) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:1175]:
```suggestion
          name: 'Generating AI images',
```

File: apps/docs/content/guides/functions/examples/amazon-bedrock-image-generator.mdx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+This guide will walk you through an example using the Amazon Bedrock JavaScript SDK in Supabase Edge Functions to generate images using the [Amazon Titan Image Generator G1](https://aws.amazon.com/blogs/machine-learning/use-amazon-titan-models-for-image-generation-editing-and-searching/) model.
+
+## Setup
+
+- In your AWS console, navigate to Amazon Bedrock and under "Request model access", select the Amazon Titan Image Generator G1 model.
+- In your Supabase project, create a `.env` file in the `supabase` directory with the following contents:
+
+```txt
+AWS_DEFAULT_REGION="<youre_region>"
```
Comment (charislam) [inline apps/docs/content/guides/functions/examples/amazon-bedrock-image-generator.mdx:18]:
```suggestion
AWS_DEFAULT_REGION="<your_region>"
```

File: apps/docs/content/guides/functions/examples/amazon-bedrock-image-generator.mdx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+import { decode } from 'npm:base64-arraybuffer'
+
+console.log('Hello from Amazon Bedrock!')
+
+Deno.serve(async (req) => {
+  prepareVirtualFile('./aws/config')
+  prepareVirtualFile('./aws/credentials')
+
+  const client = new BedrockRuntimeClient({
+    region: 'us-west-2',
```
Comment (charislam) [inline apps/docs/content/guides/functions/examples/amazon-bedrock-image-generator.mdx:61]: Do you mean to make this configurable with `AWS_DEFAULT_REGION`? (didn't see it used elsewhere so assumed that was the purpose of that env variable)

**Review by charislam — APPROVED** [review]

File: apps/docs/content/guides/functions/examples/amazon-bedrock-image-generator.mdx (hunk 3 above)
Comment (thorwebdev) [inline apps/docs/content/guides/functions/examples/amazon-bedrock-image-generator.mdx:61]:
```suggestion
    region: Deno.env.get('AWS_DEFAULT_REGION') ?? 'us-west-2',
```

Comment (thorwebdev) [reply apps/docs/content/guides/functions/examples/amazon-bedrock-image-generator.mdx]: ah yes, thanks for the catch \o/

**Pull request merged**